### ISSUE TITLE
[BUG] MQTTX Desktop UI hung on launch with Windows 11 ARM64 (surface laptop 7)

### ISSUE THREAD

**Issue by oising** [emqx/MQTTX#1803]
#### What did I do

I installed v1.11.0 and launched the app. 

#### What happened

Desktop opens and hangs continually on this view:

![image](https://github.com/user-attachments/assets/96ee5187-9e7f-4f75-a5f9-213c5f31c9c0)

note: The logo is actually spinning/animating, but the screen remains like this (waited ten minutes, rebooted, tried again with same result)

process tree looks like this:

![image](https://github.com/user-attachments/assets/b7b965b1-db77-40ab-8d6f-7ceec92a329c)

All seem to be in a wait state for userrequest according to procexp. 

#### Expected

desktop app should work :)

#### Environment

- windows 11 on arm64 (surface laptop 7)
- win build 24h2/26100

#### More detail

More detail include `log` information or screenshot

**Comment by ysfscream** [comment]
Hi, I haven't encountered this issue before. Could you please provide some logs so we can investigate further? You can find instructions on how to retrieve logs here: https://mqttx.app/docs/advanced#logs

**Comment by oising** [comment]
> Hi, I haven't encountered this issue before. Could you please provide some logs so we can investigate further? You can find instructions on how to retrieve logs here: [mqttx.app/docs/advanced#logs](https://mqttx.app/docs/advanced#logs)

Folder is empty.

**Comment by roccoder** [comment]
It's the same with my surface pro 11, hung on splashscreen. Logo is rotating waiting for something but never shows the main screen, it just stuck on boot. With x64 version everything is working correctly

**Comment by MansonStudio** [comment]
同样问题

**Comment by oising** [comment]
1.11.1 doesn't fix anything, fyi.


https://github.com/user-attachments/assets/05958164-7f83-4ff1-b81b-3a29578f5365

**Comment by ysfscream** [comment]
YES. 1.11.1 has not fixed this issue yet. But how about the x64 version?

**Comment by kic68** [comment]
I can confirm this, only rotating logo. Just installed the newest arm64 version 1.11.1 again.

**Comment by ysfscream** [comment]
Hi everyone, the issue is caused by `node-sqlite3` not having precompiled binaries for Windows ARM64, maybe, as discussed here: https://github.com/TryGhost/node-sqlite3/issues/1799. We are working on a fix and will address this in the next release. Thanks for your patience and understanding.

**Comment by ysfscream** [comment]
Hi everyone,

**Workaround:** Please try downloading the x64 version instead of ARM64 for Windows 11 ARM devices.
The SQLite compilation issue for Windows ARM64 is difficult to resolve in the current version due to node-sqlite3 dependency limitations. We need to wait for official upstream support.
I've tested locally and x64 version works fine on Windows ARM64 through emulation.
Please let us know if you encounter any issues with the x64 version. Thanks!

<img width="1844" alt="Image" src="https://github.com/user-attachments/assets/714b444a-fe1c-4e57-b29d-37ac6756cec9" />

**Comment by ysfscream** [comment]
Hi, all. Please download the 1.12.0 universal release If you’re on an ARM machine. If you have any new issues, please open a new issue. Thanks.

<img width="1113" alt="Image" src="https://github.com/user-attachments/assets/fa921f56-9cc8-4152-8115-489a5eea7da5" />
